src/index.js

```diff
@@ -53,6 +53,9 @@ export { default as isInteger } from './isInteger';
 export { default as isNotInteger } from './isNotInteger';
 export { default as isFloat } from './isFloat';
 export { default as isNotFloat } from './isNotFloat';
+export { default as isValidNumber } from './isValidNumber';
+export { default as isNotValidNumber } from './isNotValidNumber';
+export { default as isInvalidNumber } from './isNotValidNumber'; // alias of isNotValidNumber
```

Comment:
IMHO this is somehow misleading naming for `isInvalidNumber`. The name indicates that the value checked by the predicate will be number, but not a valid one e.g. NaN, Infinity.

I would imagine isInvalidNumber to have the following signature

```javascript
const isInvalidNumber = both(isNumber, isNotValidNumber);
```